doc/CustomDataContainer.md

```diff
@@ -0,0 +1,64 @@
+# CustomDataContainer
+
+
+`CustomDataContainer` renders a plug'n'play custom data card.
+
+
+## Setup
+
+Before using this component in your project, `react-i18next`, `reactstrap` and `react-hook-form` must be installed and added into the project's `package.json` file:
+
+```
+yarn add react-i18next
+yarn add reactstrap
+yarn add react-hook-form
+```
+
+At the moment works ONLY with `apiPath='seacat_auth'`
```

Comment:
@petrKavulok Again, not necessary to mention that, since it is used only within seacat-admin ui and for that you need to have `seacat_auth` proxy set in general